geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-orc/src/main/scala/org/locationtech/geomesa/fs/storage/orc/utils/OrcInputFormatReader.scala

```diff
@@ -34,24 +34,24 @@ object OrcInputFormatReader {
     var i = 0
     var col = 0
     while (i < sft.getAttributeCount) {
+      val bindings = ObjectType.selectType(sft.getDescriptor(i))
+      val reader = bindings.head match {
+        case ObjectType.GEOMETRY => col += 1; createGeometryReader(bindings(1), col - 1, col, i)
```

Comment:
i guess this needs to be outside the if to increment the col index for geom fields? maybe we could just add an else statement instead of instantiating an unneeded reader